core/src/main/java/com/google/zxing/qrcode/QRCodeWriter.java

```diff
@@ -37,6 +37,60 @@ public final class QRCodeWriter implements Writer {
 
   private static final int QUIET_ZONE_SIZE = 4;
 
+  /**
```

Comment:
I'm not against moving this, but it'd be slightly cleaner to not move the method